ee/codegen/src/__test__/nodes/templating-node.test.ts

```diff
@@ -171,4 +171,59 @@ describe("TemplatingNode", () => {
       expect(await writer.toStringFormatted()).toMatchSnapshot();
     });
   });
+
+  describe("referencing an invalid node", () => {
```

Comment:
I started the PR with this test, but am thinking it might now be redundant seeing the others that got affected